### PR TITLE
Update Sjors

### PR DESCRIPTION
**Pull request by Sjors**
* drop recurring bolt 12: it won't be part of the initial standard
* replace single bolt 12: IIUC there were some minor changes in the draft standard, so I generated a new one just in case

See 035da6e08db793eccf5d12f526c8b75f875a313f to compare commit signature.

I also have a [Lightning Address](https://lightningaddress.com) now (`sjors@bitrefill.me`), but I'd rather not override my Github Sponsor link with that.

### CODE REVIEW

**Comment by dennisreimann** [comment]
Hey Sjors, you can provide the Lightning Address separately, as `lnaddr`, see the [available options](https://github.com/dennisreimann/bitcoindevlist.com#addition-template--available-options).

**Comment by Sjors** [comment]
@dennisreimann should I set `"lightning": true` in that case?

File: donatees/sjors-provoost.json [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
   "description": "I've been contributing to Bitcoin Core on a part-time basis since mid 2017. Most of my time is spent on reviewing and testing work by others, but I also created my own pull requests. Bitcoin ACKs provides a nice overview of my activity on the repo.",
   "tags": [
     "Bitcoin Core"
   ],
-  "bolt12_single": "lno1pgsy7ur9dcs8xmm4wf3k2gryv4mx2mr0wpkk2mn5ypjx7mnpw35k7ms5pefk5mmjwvs9qun0wehk7um5rcsrwlufzdn408gvmlpgxw7rywsaq9g5a6jq6jxvapknrcucyglqevlsgrgkxx9ackp869xue05ml9fpe7cj3eddg3dt79paca9xrhhkweafr2qs4fjwn2268ud4u6lsmerarr3uu76t8068k47jh5q99rgulm07",
-  "bolt12_recurring": "lno1pg5y7ur9dcs8xmm4wf3k2gryv4mx2mr0wpkk2mn5ypkk7mn5dpk8jgrydahxzarfdahpgrjndfhhyueq2pex7an0daehgxszqgq3ugph07y3xe6hn5xdls5r80pj8gwsz52wafqdfrxwsmf3uwvzy0svk0cyqzdh9w50vty55th85pq358dvdp2j3jyyz92thwpcdhnx72tm75klhs5lk8qexx7vqhmtecapdk2hsnngzv8unzn74a7x9yq7nm8dth8q"
+  "bolt12_single": "lno1pg4y7ur9dcs8xmm4wf3k2gryv4mx2mr0wpkk2mn5ypjx7mnpw35k7m3qvehhygzndfhhyuc5pefk5mmjwvs9qun0wehk7um5rcsrwlufzdn408gvmlpgxw7rywsaq9g5a6jq6jxvapknrcucyglqevlsgqgf4k7rc5runmkuex8ue6jrj0r9jszltusclp4yd0z4mekwa7k8dfskd2ltdlfmcdnm6f0sv5jsrs6t3vkql8nav0vl6w4l44gmaf4c",
+  "lnaddr": "sjors@bitrefill.me"
```
Comment (Sjors) [inline donatees/sjors-provoost.json:13]: I might update this to something self hosted later.

**Review by dennisreimann — APPROVED** [review]

**Comment by dennisreimann** [comment]
Thanks for the heads-up, I'll change the default setting for pr merges.

**Pull request merged**